tema5/src/main/java/ro/sci/carrental/repository/CustomerDAO.java

```diff
@@ -0,0 +1,84 @@
+package ro.sci.carrental.repository;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ro.sci.carrental.domain.car.Car;
+import ro.sci.carrental.domain.car.Category;
+import ro.sci.carrental.domain.car.GearBoxType;
+import ro.sci.carrental.domain.customer.Address;
+import ro.sci.carrental.domain.customer.Customer;
+import ro.sci.carrental.service.*;
+
+import java.sql.*;
+
+/**
+ * Created by Ivett on 11-Aug-17.
+ */
+public class CustomerDAO {
+    CustomerRepository<Customer> customerRepository = new CustomerRepositoryImpl();
+    CustomerService<Customer> customerService =  new CustomerServiceImpl(customerRepository);
+    private static final Logger LOGGER = LoggerFactory.getLogger("CustomerDB");
+
+    public void printAllCustomer() {
+        try (Connection conn = newConnection("postgresql",
```

Comment:
asta cu newConnection() iar ar putea fi extrasa in acel AbstractRepository, ca sa nu cari peste tot parametri ;)